### ISSUE TITLE
Message view tool

### ISSUE THREAD

**Issue by jdotcms** [dotCMS/core#21253]
Currently the user has the ability to send message to the clients by using websockets and the message actionlet.

However on the **velocity script actionlet** would be very useful to have the ability to send message to the client depending on some condition or another.

Create a view tool that allow to the user to send messages to the current user such as 
```
#if ( some-condition)
$workflow.abortProcessor()
$messagetool.sendInfo("The workflow has been cancelled b/c ....")

#end
```

It will has 

- sendInfo
- sendWarn
- sendError

STR:
1) Navigate to Workflows
2) On the publish step and add velocity script actionlet with the code:
`$messagetool.sendInfo("Published")`
3) Publish any content

**Comment by jdotcms** [comment]
Examples how to use it

```
$messagetool. sendInfo('Hello');

$messagetool.sendInfo('Hello', 10);

$messagetool.sendInfo('Hello', 10, ['dotcms.org.1']);
```

The first example, just shows the message 'Hello' (by default 5 seconds to all users)
The second increase to 10 seconds the life time and finally the third one sends the message only to an one or more userids.

It looks such as:
![sendInfo](https://user-images.githubusercontent.com/19578350/143134261-2a59729b-b689-4f5b-9880-9fabb1600691.jpg)

**Comment by jdotcms** [comment]
```

$messagetool.sendWarning('Hello');
$messagetool.sendWarning('Hello', 10);
$messagetool.sendWarning('Hello' 10, ['dotcms.org.1']);
```
![warning](https://user-images.githubusercontent.com/19578350/143134423-6ae45499-7066-4536-8df5-0c9d70ee2692.jpg)

**Comment by jdotcms** [comment]
```
$messagetool.sendError('Hello');
$messagetool.sendError('Hello', 10);
$messagetool.sendError('Hello' 10, ['dotcms.org.1']);
```
![error](https://user-images.githubusercontent.com/19578350/143134619-5310d53d-4269-490f-8fc2-23e94295b942.jpg)

**Comment by jdotcms** [comment]
```

$messagetool.sendSuccess('Hello');
$messagetool.sendSuccess('Hello', 10);
$messagetool.sendSuccess('Hello' 10, ['dotcms.org.1']);
```
![info](https://user-images.githubusercontent.com/19578350/143134841-d51e7b39-efb3-4529-b36c-54f669ac6a4f.jpg)

**Comment by bryanboza** [comment]
Fixed, tested on release-22.02 // Docker // FF and now we are able to send this kind of messages without problems
